<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>datasetsUI</class>
 <widget class="QWidget" name="datasetsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>划分数据集</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="dataBackWdt" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>4</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>4</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="data1Lr">
            <property name="text">
             <string>---划分数据集---</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="closeBtn">
            <property name="minimumSize">
             <size>
              <width>17</width>
              <height>17</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>17</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="data2Lr">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>验证集比例</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="ratioEt">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="text">
                 <string>0.2</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="data6Lr">
              <property name="text">
               <string>默认值为0.2即20%的数据作为验证集</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="data3Lr">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>图片路径</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="picPathEt">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>36</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="picChBtn">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>36</height>
               </size>
              </property>
              <property name="text">
               <string>更改</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="data4Lr">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>标签路径</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lablePathEt">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>36</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="labelChBtn">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>36</height>
               </size>
              </property>
              <property name="text">
               <string>更改</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="data5Lr">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>输出目录</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="datasetPathEt">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>36</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="outChBtn">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>36</height>
               </size>
              </property>
              <property name="text">
               <string>更改</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="proWdt" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="msgLr">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QProgressBar" name="progressBar">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>14</height>
                </size>
               </property>
               <property name="value">
                <number>24</number>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="format">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPushButton" name="dataStartBtn">
              <property name="minimumSize">
               <size>
                <width>134</width>
                <height>36</height>
               </size>
              </property>
              <property name="text">
               <string>开始划分</string>
              </property>
              <property name="icon">
               <iconset resource="dubhe.qrc">
                <normaloff>:/dubhe/pic/start.png</normaloff>:/dubhe/pic/start.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="dubhe.qrc"/>
 </resources>
 <connections/>
</ui>
